pylace/README.md

```diff
@@ -31,9 +31,11 @@ file. The following lace functions are supported:
 
 ```python
 import lace
+import polars as pl
 
 # The required files can be found here: https://github.com/promised-ai/lace/tree/master/pylace/lace/resources/datasets/satellites
-engine = lace.Engine(data_source="data.csv", codebook="codebook.yaml")
+df = pl.read_csv("./data.csv")
+engine = lace.Engine.load((df, codebook_builder="codebook.yaml")
```

Comment:
I think the arg name should stay `codebook`.

Reply:
Agreed. Fixed.